### PR TITLE
get rid of can-util

### PR DESCRIPTION
**Pull request by cherifGsoul**
for https://github.com/canjs/canjs/issues/4218

### CODE REVIEW

File: bubble.js [hunk 1]
```diff
@@ -23,9 +23,11 @@
 
 var canEvent = require('can-event-queue/map/map');
 
-var makeArray = require('can-util/js/make-array/make-array');
 var canReflect = require('can-reflect');
-var isEmptyObject = require('can-util/js/is-empty-object/is-empty-object');
+var makeArray = canReflect.toArray;
```
Comment (chasenlehara) [inline bubble.js]: I would just use canReflect.toArray where you need it.

File: bubble.js [hunk 2]
```diff
@@ -23,9 +23,11 @@
...
+var isEmptyObject = function(value) {
+	return canReflect.size(value) === 0;
+};
```
Comment (chasenlehara) [inline bubble.js]: This is only used once in the file, I would put the check inline.

File: bubble.js (hunk 1 above)
Comment (cherifGsoul) [reply bubble.js]: Fixed!

File: bubble.js (hunk 2 above)
Comment (cherifGsoul) [reply bubble.js]: Fixed!

File: can-map.js [hunk 3]
```diff
@@ -22,19 +22,18 @@ var Construct = require('can-construct');
...
+var singleReference = require('can-single-reference');
 
 var namespace = require("can-namespace");
-var dev = require("can-util/js/dev/dev");
+var dev = require("can-log/dev/dev");
 var CID = require("can-cid");
-var deepAssign = require("can-util/js/deep-assign/deep-assign");
 //var isFunction = require("can-util/js/is-function/is-function");
-var assign = require("can-util/js/assign/assign");
+var assign = require("can-assign");
```
Comment (chasenlehara) [inline can-map.js]: It worries me a little bit that the tests are passing but this module isn’t in package.json

Comment (cherifGsoul) [reply can-map.js]: Thanks, fixed!

**Pull request merged**